backend/src/org/commcare/suite/model/graph/Graph.java

```diff
@@ -141,26 +141,35 @@ private void evaluateConfiguration(Configurable template, ConfigurableData data,
     }
 
     /*
-     * Helper for evaluate. Looks at a single series.
+     * Helper for evaluate. Looks at all series.
      */
     private void evaluateSeries(GraphData graphData, EvaluationContext context) {
         try {
             for (XYSeries s : mSeries) {
+                Hashtable<String, Text> expandableConfiguration = new Hashtable<>();
+                for (Enumeration<String> e = s.getExpandableConfigurationKeys(); e.hasMoreElements();) {
```

Comment:
I think that type annotations inside of for-each loops break J2ME as well.
you'll need `for(Enumberation e = ...)`